src/AppBundle/Resources/views/Layouts/moj_template.html.twig

```diff
@@ -104,6 +104,12 @@
     </div>
 
     <div class="govuk-width-container {% if env == 'admin' %}opg-width-container--wider{% endif %}">
+        {% block detailsBanner %}
+            {% include 'AppBundle:Org/ClientProfile:_detailsBanner.html.twig' with {
+            'report': report is defined ? report : null
+            } %}
+        {% endblock %}
+
```

Comment:
Can you test this with a prof/pa report? I think there was a reason I put it in the application template? Possibly not, just a niggling thought

Reply:
253a9d4 addresses this, ensuring the lay positioning is relative to the correct element.